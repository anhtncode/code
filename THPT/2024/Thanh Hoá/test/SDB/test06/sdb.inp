64
7 2635
5 462
3 4558
6830 9308
3413 8894
3097 6293
3926 3955
5422 6802
1 5137
7 9102
5 470
2182 4884
10 4817
8 8524
8631 9576
255 2324
7680 7934
3334 3621
6 3985
887 8518
3 8853
6014 8268
1147 4932
7878 8445
44 3444
8 1309
2592 8627
6 7224
5 4039
782 8520
2077 8936
10 9945
2148 9365
8574 9205
3132 5081
1 2227
1096 2400
5 7446
3958 7105
7 3016
1452 9064
1723 1894
8 3870
4822 5064
1672 9935
8 4244
2 7433
5 1556
5520 9041
6 1052
2961 8167
2206 7274
1426 5933
5 9980
1629 9206
5 2715
468 8133
1623 2233
8 6709
3974 6143
2599 4959
9 3203
10 6494
1 2864
