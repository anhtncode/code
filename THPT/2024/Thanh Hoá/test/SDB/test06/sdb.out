6
3
7
2
4
2
0
1
8
11
3
3
8
10
1
3
0
0
7
7
11
2
4
0
6
4
5
9
7
7
6
11
6
1
2
5
1
9
2
6
7
0
7
0
6
7
9
5
3
3
4
4
4
11
6
6
7
0
9
1
3
6
8
6
